assembly/compliance-tool/pom.xml

```diff
@@ -62,6 +62,9 @@
                     <include>**/*.md.vm</include>
                 </includes>
             </resource>
+            <resource>
+                <directory>${project.basedir}/src/main/resources</directory>
+            </resource>
         </resources>
     </build>
 
```

Comment:
In order to work correctly, the logging requires some dependencies from the Apache log4j project. I checked the dependency tree, and the dependencies in question are actually present, but they are pulled in transitively via the spdx-tools dependency. I would recommend specifying them explicitly in the pom. This can be done by adding the following dependencies (analogously to the pom of the cli module):

```
        <dependency>
            <groupId>org.eclipse.sw360.antenna</groupId>
            <artifactId>cli-frontend-stub</artifactId>
            <version>${project.version}</version>
        </dependency>
        <dependency>
            <groupId>org.apache.logging.log4j</groupId>
            <artifactId>log4j-core</artifactId>
        </dependency>

```